src/utils/getNodeIconName.js

```diff
@@ -0,0 +1,10 @@
+/**
+ * Simple resolver for icon naming.
+ *
+ * Reconcile protocol "venue" naming with Network-Canvas-UI (svg) "place"
```

Comment:
Can we just update the svg in Network-Canvas-UI if we prefer "venue", or is there a reason for having different naming schemes?

Reply:
I'd personally prefer to use one or the other... I'll open an issue on UI if others agree that 'venue' is better.

Reply:
Hm. so something has gone partially awry here. 

So, the node types (and colors) are supposed to have one canonical point of definition: the variable registry section of the protocol file:

``` JSON
"variableRegistry": {
  "node": {
    "person": {
      "label": "Person",
      "color": "coral",
      "displayVariable": "nickname",
      "variables": { ... }
    },
  },
}
```
As you can see, this defines both the node type and the corresponding node color, with reference to the existing palette colors (which the node component itself should accept as a parameter: `<Node color="coral" />`). Where no color is defined, we don't have specified behavior yet. A given prompt on an interface will refer to a node type, and the interface should look up the corresponding meta-data (such as node color) here.

To handle the icons, we should introduce an additional property here, something like `iconVariant`. Individual interfaces can then use `getNodeIconName.js` to resolve this. The Name Generator and other interfaces that show a new node button will need to have this functionality added.

I might have misunderstood your discussion, so sorry if I've just confused matters.

Reply:
- [x] look up color in registry
- [x] look up iconVariant in registry